code/modules/modular_computers/terminal/terminal_commands.dm

```diff
@@ -577,6 +577,9 @@ Subtypes
 
 /datum/terminal_command/com/proper_input_entered(text, mob/user, datum/terminal/terminal)
 	// If the user is unskilled, call a random method
+	if(!terminal.computer.get_network_status(NET_FEATURE_SYSTEMCONTROL))
```

Comment:
maybe stick it in 
```
/datum/terminal_command/proc/parse(text, mob/user, datum/terminal/terminal)
```
there's already some network access checks in it
can just set required function as var on command like programs do and check in one place then

Reply:
Done so.